### ISSUE TITLE
Clicking the Children Button of an Element in Inspector Crashes Cogent

### ISSUE THREAD

**Issue by DapperMongoose** [cogentcore/core#1320]
### Describe the bug

There's some sort of edit icon present for the Children field of an element in the inspector.  When clicking it Cogent will crash (see slightly redacted crash log)

[crash_2024-11-17_10-13-21.txt](https://github.com/user-attachments/files/17791680/crash_2024-11-17_10-13-21.txt)

### How to reproduce

Run the example minimal reproduction
Open inspector
Click on an element (Submit was used for the included log)
Click on the button in the Children field (1 nodes)

### Example code

```go
package main

import (
	"cogentcore.org/core/core"
	"cogentcore.org/core/events"
	"fmt"
)

func main() {
	b := core.NewBody()
	core.NewText(b).SetText("Minimal Reproduction")
	textField := core.NewTextField(b)
	textField.StartFocus()

	btn := core.NewButton(b).SetText("Submit")

	btn.OnClick(func(e events.Event) {
		core.MessageSnackbar(b, fmt.Sprintf("button click, text content: %v", textField.Text()))
		textField.SetText("")
	})

	//btn.SetKey(keymap.Enter)
	//btn.SetShortcut(keymap.Enter.Chord())
	btn.SetShortcut("Command+S")

	b.RunMainWindow()
}
```

### Platform

Linux

**Comment by kkoreilly** [comment]
Thank you for reporting this and providing a simple reproducible example. I can reproduce this and will fix it soon.

**Comment by kkoreilly** [comment]
I fixed this in #1336, so that button does not crash anymore.
